_contributors-app/palash25.md

```diff
@@ -50,11 +50,10 @@ project:
      written using Requests and BeautifulSoup.
 
 volunteer_experience: |
- - Most of my time is spent hacking on open source. I have contributed to coala, duckduckgo, Kinto, OpenGenus,Mozilla and the Ethereum foundation.
+ - I am the head of the Programming Society and the club captain of the Mozilla Campus Club of IIIT-Bh. Together we try to guide the first year students in getting started with software development.
 
- - I also like to write articles and have written for opensource.com and geeksforgeeks.org in the past. I also maintain a list of links to all my articles just in case you are interested.
+ - Most of my time is spent hacking on open source. I have made contributions to coala, duckduckgo, Kinto, OpenGenus, Mozilla and the Ethereum foundation. Moreover, I am fond of writing articles. I have prior experience in content writing for opensource.com and geeksforgeeks.org. For those interested, I maintain a list of links to all my articles.
```

Comment:
can you add this as a link to the last word of this point i.e `[articles](https://github.com/palash25/articles/)` ?

Reply:
The rest LGTM :+1:

Reply:
Could you tell me what the "remote name" is? I'll do a force push.

Reply:
The remote repo is the original repo -- https://github.com/p-society/p-society.github.io.git

Reply:
Could you please google that first and then ask me if you still don't understand?

Reply:
Sorry, got it!

Reply:
I actually did. I couldn't understand so I asked. Sorry.

Reply:
remote_name is the name given to the repository url. Do `git remote -v` and tell me what you see.

Reply:
origin	https://github.com/itsmepiyush2/p-society.github.io.git (fetch)
origin	https://github.com/itsmepiyush2/p-society.github.io.git (push)

Reply:
that means origin is your remote name. You will also need to add an upstream remote but we can do that after this PR. The upstream will point to this repo i.e. `p-society/p-society.githu.....`